test_inputs/.gitignore

```diff
@@ -0,0 +1 @@
+hello_world/hello_world1
```

Comment:
Not `hello_world2` and others?

Reply:
Well spotted.

Reply:
Actually we don't need a `.gitignore` since I made all outputs go into `target` which is ignored in its entirety by the top-level `.gitignore`.